tests/test_api_async_example.py

```diff
@@ -0,0 +1,72 @@
+"""
+
+API EXAMPLE TEST
+1. Add new car - POST request(without url_params)
+2. Get all cars - GET request(without url_params)
+3. Verify car count
+4. Update newly added car details -PUT request
+5. Get car details -GET request(with url_params)
+6. Register car - POST request(with url_params)
+7. Get list of registered cars -GET
+8. Verify registered cars count
+9. Delete newly added car -DELETE request
+"""
+
+import asyncio
+import os
+import sys
+import pytest
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from conf import api_example_conf
```

Comment:
@shivahari I ran test and noticed you adding new car as figo and verifying swift as a new car. Is configurations are correct? Do you missed to commit config file changes?

Reply:
This is intentional @rohandudam. These are two separate validations.
Adding a car and validating if the car is added is not a right test scenario for `Async` test ❌ . We will hit `race-condition` if we add a car and try to validate if the car is added. With `Async Task Group` the tasks are not executed in a sequential order, so when we run the test the scenario to check if the new car is added might at times be executed before the add car scenario completes. This will result in a `Failure`.

Reply:
I have modified the test to:
1. Update the `module-dicstring`
2. Add useful comments
3. Use appropriate variable names

I have not added a statement to print the list of cars after running a `GET` against `/cars`, I had used statements to print the test data only, will skip printing the list of cars output hence.